src/trans_unit.jl

```diff
@@ -6,6 +6,8 @@ Parse the given source file and the translation unit corresponding to that file.
 mutable struct TranslationUnit
     ptr::CXTranslationUnit
     idx::Index
+    source_filename::String
```

Comment:
this might break someone's custom generator which uses `TranslationUnit` exported by Clang.jl.

Reply:
I've reverted the change in af62eff, but how would this break a custom generator? :thinking:

Reply:
very unlikely, but possible if they've extended a `source_filename` field via `getproperty`...